python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -78,6 +78,17 @@ class AutoMaterializeAssetEvaluation(NamedTuple):
     num_requested: int
     num_skipped: int
     num_discarded: int
+    run_ids: Set[str] = set()
+
+    def with_run_ids(self, run_ids: Set[str]) -> "AutoMaterializeAssetEvaluation":
```

Comment:
nit: I think we can use the built-in `._replace()` on the NamedTuple class, i.e. `return self._replace(run_ids=run_ids)`